### DISCUSSION TITLE
PINND

### DISCUSSION THREAD

**Discussion by Boos4721** [Boos4721/openwrt#140] [Announcements]
**I am being quarantined because of the COVID-19 and will be updated in a few days .**
Because just passing through Henan and Wuhan. ):

**Comment by he110777** [comment]
omg，注意身体。

**Comment by zjwxh** [comment]
大佬，多多保重，你的源码给我们帮助很大！

**Comment by devnakx** [comment]
thx for your works, take care yourself.

**Comment by online2311** [comment]
照顾好自己,我已经快要解除隔离了。
